extensions/binding/org.eclipse.smarthome.binding.tradfri.test/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,25 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name:  Tradfri Binding Tests
+Bundle-SymbolicName: org.eclipse.smarthome.binding.tradfri.test;singleton:=true
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-Version: 0.9.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Fragment-Host: org.eclipse.smarthome.binding.tradfri
+Import-Package: org.eclipse.smarthome.binding.tradfri,
+ org.eclipse.smarthome.binding.tradfri.handler,
+ org.eclipse.smarthome.core.common.registry,
+ org.eclipse.smarthome.core.events,
+ org.eclipse.smarthome.core.thing.util,
+ org.eclipse.smarthome.test.java,
+ org.eclipse.smarthome.test.storage,
+ org.hamcrest;core=split,
+ org.junit;version="4.0.0",
```

Comment:
Okay and for my understanding why? I keep telling people they should not add version numbers I don't see why there should be an exception here, is that maybe an error?